addons/l10n_hr/data/l10n_hr_chart_data.xml

```diff
@@ -10,8 +12,8 @@
             <field name="bank_account_code_prefix">101</field>
             <field name="cash_account_code_prefix">102</field>
             <field name="transfer_account_code_prefix">1009</field>
-            <field name="currency_id" ref="base.HRK"/>
-            <field name="code_digits">0</field>
+            <field name="currency_id" ref="base.EUR"/>
```

Comment:
Will be vaild starting 1.1.2023.
HRK should stay for at least one or two odoo releases before removing it from list of currencies